pkg/totalaccountwatcher/totalaccountwatcher_test.go

```diff
@@ -47,7 +47,7 @@ func TestAccountWatcherCreation(t *testing.T) {
 		// after mocks is defined
 		defer mocks.mockCtrl.Finish()
 
-		totalAccountWatcher := NewTotalAccountWatcher(mocks.fakeKubeClient, mocks.mockAWSClient, 10)
+		totalAccountWatcher := newTotalAccountWatcher(mocks.fakeKubeClient, mocks.mockAWSClient, 10)
```

Comment:
Does this still work now that it's not exported? I guess its in the same pkg

Reply:
Yeah, it appears to work to me, as you said, in the same package.